GDJS/Runtime/events-tools/soundtools.js

```diff
@@ -87,6 +87,11 @@ gdjs.evtTools.sound.getSoundOnChannelPitch = function(runtimeScene, channel) {
     return sound ? sound.rate() : 1;
```

Comment:
I think you may have forgotten to update this line to use `getRate`? :)

Reply:
Absolutely!, fixed :)